exercises/proverb/canonical-data.json

```diff
@@ -103,6 +103,21 @@
                 "For want of a soldier the battle was lost.",
                 "And all for the want of a pin."
             ]
+        },
+        {
+          "uuid": "4da2d184-a631-4f31-b770-52cccca54dd1",
+          "description": "five pieces modernized",
```

Comment:
the reason this test case is important is because there is an element beginning with a vowel; the number of pieces is unimportant. Therefore, the description should reflect this and say "item beginning with vowel" or similar.

Reply:
Yeah, def
I'll update the comment.